src/web/core/src/main/java/org/geoserver/web/data/resource/BasicResourceConfig.java

```diff
@@ -360,11 +356,16 @@ private String getActualNativeSRSCode(ResourceInfo resourceInfo) {
         return "";
     }
 
-    private List<String> getOtherSRSFromWFSOrWMS(ResourceInfo resourceInfo) {
+    private List<String> getOtherSRSFromWFSOrWMSOrWMTS(ResourceInfo resourceInfo) {
         // first check if its WFS-NG
         List<String> otherSRS = DataStoreUtils.getOtherSRSFromWfsNg(resourceInfo);
```

Comment:
I'd go and check if it's a FeatureTypeInfo here.

Reply:
fixed